plugins/Scheduler/test.py

```diff
@@ -74,6 +74,13 @@ def test00RemoveZero(self):
         timeFastForward(5)
         self.assertNoResponse(' ', timeout=1)
 
+    def testRemind(self):
+        self.assertNotError('scheduler remind 5 testRemind')
+        timeFastForward(3)
```

Comment:
Make it 4; because 3+2 is too close to 5, so `assertResponse` might run just before it's scheduled. With 4+2 it's sure it will run after the reminder is scheduled